core/src/main/java/org/apache/iceberg/util/JsonUtil.java

```diff
@@ -98,6 +98,12 @@ public static <T> T parse(String json, FromJson<T> parser) {
     }
   }
 
+  public static JsonNode get(String property, JsonNode node) {
+    Preconditions.checkArgument(
+        node.hasNonNull(property), "Cannot parse missing object '%s'", property);
```

Comment:
The error message should be `"Cannot parse missing field: %s"`. We generally prefer context after `:` rather than quoting values for clarity.

Reply:
updated